src/MainMenu.cpp

```diff
@@ -0,0 +1,54 @@
+#include <D3DX8.h>
+#include <windows.h>
+
+#include "MainMenu.hpp"
+#include "Supervisor.hpp"
+
+#pragma optimize("s", on)
+ZunResult MainMenu::BeginStartup()
+{
+    D3DXVECTOR3 vector3Ptr; // we have to add Ptr,
+                            // because otherwise it gets 0.7% less on decomp.me for some reason
+    DWORD time;
+    int i;
+
+    if (LoadTitleAnm(this) != ZUN_SUCCESS)
+    {
+        g_Supervisor.curState = 4;
+        return ZUN_ERROR;
+    }
+    else
```

Comment:
You can drop the else entirely here, since the if above ends with a return. Shouldn't cause any codegen changes.